CMakeLists.txt

```diff
@@ -78,11 +78,14 @@ set_target_properties(${LIBRARY_NAME}  PROPERTIES VERSION ${${VARS_PREFIX}_VERSI
                                        SOVERSION ${${VARS_PREFIX}_VERSION}
                                        PUBLIC_HEADER "${folder_header}")
 
-target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>"
-                                           "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/${${VARS_PREFIX}_INSTALL_INCLUDEDIR}>"
-                                           ${YARP_INCLUDE_DIRS}
-                                           ${iDynTree_INCLUDE_DIRS}
-                                           ${wholeBodyInterface_INCLUDE_DIRS})
+target_include_directories(${LIBRARY_NAME} PUBLIC "$<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include/yarpWholeBodyInterface>"
```

Comment:
Yes.. you are right.
So.. I modified the tests.
If this compiles, we can restart the travis on the super build and see what happens.
Otherwise I'll try to modify the inclusions in the public headers.